packages/driver/src/cy/commands/navigation.ts

```diff
@@ -1160,7 +1162,7 @@ export default (Commands, Cypress, cy, state, config) => {
           // tell our backend we're changing origins
           // TODO: add in other things we want to preserve
           // state for like scrollTop
-          let s: Record<string, any> = {
+          let s: RunState = {
```

Comment:
Almost wish `RunState` was declared with all properties required, and references would just be a `Partial<RunState>`. Almost looks like the whole interface is partial?